guacamole/src/main/frontend/src/translations/ca.json

```diff
@@ -428,6 +541,7 @@
         "FIELD_OPTION_FONT_SIZE_60"    : "60",
         "FIELD_OPTION_FONT_SIZE_72"    : "72",
         "FIELD_OPTION_FONT_SIZE_96"    : "96",
+        "FIELD_OPTION_FONT_SIZE_EMPTY" : "",
```

Comment:
No need to include this, as it requires no translation.

Reply:
Removed